### ISSUE TITLE
Node automatically goes from MAINTENANCE state to ONLINE after an epoch

### ISSUE THREAD

**Issue by fyrchik** [nspcc-dev/neofs-node#1900]
```
> bin/neofs-cli -c cfg1.yml netmap nodeinfo | head -n2
key: 022bb4041c50d607ff871dec7e4cd7778388e0ea6849d84ccbd9aa8f32e16a8131
state: online

> bin/neofs-cli -c cfg1.yml control set-status --status maintenance
Network status update request successfully sent.
... TICK

> bin/neofs-cli -c cfg1.yml netmap nodeinfo | head -n2
key: 022bb4041c50d607ff871dec7e4cd7778388e0ea6849d84ccbd9aa8f32e16a8131
state: maintenance
... TICK

> bin/neofs-cli -c cfg1.yml netmap nodeinfo | head -n2
key: 022bb4041c50d607ff871dec7e4cd7778388e0ea6849d84ccbd9aa8f32e16a8131
state: online
```
I believe this is a bug, we should exit MAINTENANCE manually.

**Comment by cthulhu-rider** [comment]
Boostrap routine corrupts the state
https://github.com/nspcc-dev/neofs-node/blob/1406d096a21309fb8beeeb7a4a31f51b2ce5610a/cmd/neofs-node/config.go#L837-L846

**Comment by fyrchik** [comment]
I think we may also have a separate configuration in IR for the number epochs we wait before node in MAINTENANCE is removed from the network map.

**Comment by vkarak1** [comment]
I have reproduced the problem, reopen issue

[Attached logs](https://github.com/nspcc-dev/neofs-node/files/9838236/21.10.MaintOnline.zip)

NeoFS Storage node
Version: v0.33.0-27-g7d456cb4
GoVersion: go1.18.4

**Comment by anatoly-bogatyrev** [comment]
Issue has been reproduced in the HW-cluster. On the second epoch tick state of the node has been returned to the ONLINE status in the network map. At the same time, the behavior remained as if node is in MM. Version - latest master (v0.33.0-62-g713fdab1).

**Comment by vkarak1** [comment]
I have reproduced it with:
NeoFS Storage node
Version: v0.33.0-78-g846ff515
GoVersion: go1.18.4

Please find logs and steps below:
[tick epoch in MM mode.zip](https://github.com/nspcc-dev/neofs-node/files/9877348/tick.epoch.in.MM.mode.zip)

**Comment by fyrchik** [comment]
Closed via #1993
